### PR TITLE
Added http error code and error status for responseStreamChunk error

### PR DESCRIPTION
**Pull request by kdima**
This is a fix for #58

### CODE REVIEW

File: runtime/handler.go [hunk 1]
```diff
@@ -16,6 +17,13 @@ type responseStreamChunk struct {
 	Error  string        `json:"error,omitempty"`
 }
 
+type responseStreamError struct {
+	GrpcCode   int    `json:"grcp_code, omitempty"`
+	HttpCode   int    `json:"http_code, omitempty"`
```
Comment (yugui) [inline runtime/handler.go]: Golint error.  Should be `HTTPCode`.

File: runtime/handler.go [hunk 2]
```diff
@@ -16,6 +17,13 @@ type responseStreamChunk struct {
...
+	Message    string `json:"message, omitempty"`
+	HttpStatus string `json:"http_status, omitempty"`
```
Comment (yugui) [inline runtime/handler.go]: ditto.

**Comment by yugui** [comment]
I like the direction, but fix some errors.

* Two golint errors.
* Build error.  This error was overlooked because `.travis.yaml` of the repository was wrong.  So I have just fixed `.travis.yaml` in the master branch.

**Comment by kdima** [comment]
I have fixed the mentioned issues. Could you please have an other look?

**Comment by yugui** [comment]
LGTM.  Thank you for your contribution.

**Pull request merged**